packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -769,7 +797,7 @@ contract GuardiansRegistry is ControlledRecoverable, IGuardiansRegistry, IERC900
     * @param _amount Amount of tokens to be added to the activation locked amount of the guardian
     */
     function _lockActivation(address _guardian, address _lockManager, uint256 _amount) internal {
-        bool isAllowed = _isLockManagerAllowed(_lockManager) || _isLockManagerAllowed(ANY_ENTITY);
+        bool isAllowed = _isLockManagerAllowed(_lockManager) || _isLockManagerAllowed(ANY_ENTITY) || _guardian == _lockManager;
```

Comment:
Not sure I understand this change; would we ever expect a guardian to use themselves as a lock manager (even if the guardian was a contract)?

Reply:
Rolling it back